templates/index.html

```diff
@@ -25,111 +25,147 @@
-            <td>
-              <p>Documentation for Landscape.</p>
-              <p><a href="/landscape/en">Read the Landscape documentation&nbsp;&rsaquo;</a></p>
-            </td>
-          </tr>
-          <tr>
-            <td><a href="https://conjure-up.io" class="p-link--external">Conjure-up</a></td>
-            <td>
-              <p>Documentation for Conjure-up.</p>
-              <p><a href="https://docs.conjure-up.io/en">Read the Conjure-up documentation&nbsp;&rsaquo;</a></p>
-            </td>
-          </tr>
-          <tr>
-            <td><a href="/phone/en">Phone documentation</a></td>
-            <td>
-              <p>Documentation for Phone.</p>
-              <p><a href="/phone/en">Read the Phone documentation&nbsp;&rsaquo;</a></p>
-            </td>
-          </tr>
-          <tr>
-            <td><a href="https://github.com/canonicalltd/documentation-builder" class="p-link--external">documentation-builder</a></td>
-            <td>
-              <p>A tool for generating documentation HTML from Markdown in the standard Canonical format.</p>
-              <p><a href="/documentation-builder">Read the documentation-builder documentation&nbsp;&rsaquo;</a></p>
-            </td>
-          </tr>
-          <tr>
-            <td>Style Guide</td>
-            <td>
-              <p>The style guide for documentation for projects.</p>
-              <p><a href="/styleguide/en">Read the Style Guide&nbsp;&rsaquo;</a></p>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <h2>Product documentation</h2>
+    </div>
+    <div class="row">
+      <ul class="p-matrix">
+        <li class="p-matrix__item">
+          <a href="https://docs.maas.io/2.3/en/">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/0de4fcd5-logo-maas-icon.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://docs.maas.io/2.3/en/">MAAS&nbsp;</a></h4>
+            <p class="p-matrix__desc">Create a bare-metal cloud with Metal as a Service for IPAM and provisioning</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="https://jujucharms.com/docs/stable/getting-started">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/31c507a5-logo-juju-icon.svg" alt="Icon">
+          </a>
+          <div class="p-matrix__content">
+            <h4 class="p-matrix__title"><a class="p-link" href="https://jujucharms.com/docs/stable/getting-started">JUJU&nbsp;</a></h4>
+            <p class="p-matrix__desc">Model-driven multi-cloud operations for applications with big data, k8s and openstack solutions</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <a href="https://docs.snapcraft.io">
+            <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/82dc2022-snapcraft-primary-icon.svg" alt="Icon">
```

Comment:
@Greggless @spencerbygraves Do we have a snapcraft logo with a dark background for light pages?